ts/BUILD.typescript

```diff
@@ -70,9 +70,9 @@ npm_package(
 npm_link_package(
     name = "node_modules/@bazel/worker",
     src = "npm_at_bazel_worker",
-    deps = [
-        ":node_modules/google-protobuf"
-    ],
+    deps = {
+        ":node_modules/google-protobuf/store/0.0.0": "",
```

Comment:
So confusing. Maybe the version shouldn't appear in this case?

Reply:
Maybe, but I like the consistency of always having a version with 0.0.0 being a default for 1p deps; you can override to any version you want when you link